integration-cli/docker_cli_commit_test.go

```diff
@@ -116,9 +116,9 @@ func (s *DockerSuite) TestCommitChange(c *check.C) {
 		"--change", "ENV test 1",
 		"--change", "ENV PATH /foo",
 		"--change", "LABEL foo bar",
-		"--change", "CMD [\"/bin/sh\"]",
+		"--change", `"CMD [""/bin/sh""]"`,
```

Comment:
Ya, `StringSliceVar` is the one that uses csv, and `ListOpts` does not. So changing the type back to the old type should fix this issue and you'll be able to revert this change to the test.